capstone/src/main/webapp/about.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="Christopher Mohri, Daniel Ortega and Oluwadamilare Alamutu">
+    <meta name="generator" content="Jekyll v3.8.5">
+    <title>Perspective Hub</title>
+    <link rel="apple-touch-icon" sizes="180x180" href="assets/apple-touch-icon.png">
+    <link rel="icon" type="image/png" href="assets/favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="assets/favicon-16x16.png" sizes="16x16">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" 
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link href="cover.css" rel="stylesheet">
+  </head>
+  <body class="text-center">
+    <div class="cover-container d-flex w-100 h-100 p-3 mx-auto flex-column">
+      <header class="masthead mb-auto">
+        <div class="inner">
+          <div class="text-center">
+            <img src="assets/perspective_horizontal_color.png" class="rounded">
+          </div>
+          <h3 class="masthead-brand">Perspective Hub</h3>
+          <nav class="nav nav-masthead justify-content-center">
+            <a class="nav-link" href="index.html">Home</a>
+            <a class="nav-link" href="text_analyzer.html">Text Analyzer</a>
+            <a class="nav-link" href="image_analyzer.html">Image Analyzer</a>
+            <a class="nav-link" href="document_analyzer.html">Document Analyzer</a>
+            <a class="nav-link" href="https://support.perspectiveapi.com/s/" target="_blank">Support</a>
+            <a class="nav-link active" href="about.html">About</a>
+          </nav>
+        </div>
+      </header>
+      <main role="main">
+        <section class="jumbotron text-center">
+          <div class="container">
+            <h1>The Creators Of Perspective Hub</h1>
+            <p class="lead text-muted">A motivated team of Google STEP interns who took on the task of making Perspective API accessible to all</p>
+            <p>
+              <a href="https://medium.com/the-false-positive" class="btn btn-primary my-2" target="_blank">Perspective Blog</a>
+            </p>
+          </div>
+        </section>
+        <div class="album py-5 bg-transparent">
+          <div class="container">
+            <div class="row">
+              <div class="col-md-4">
+                <div class="card mb-4 shadow-sm">
+                  <img class="rounded mx-auto d-block" src="assets/olu.png" alt="Olu Alamutu" style="max-width:100px;max-height:225px;"></img>
```

Comment:
nit: This `style="max-width:100px;max-height:225px;"` is used in a few `<img`> tags in this file. Consider defining a CSS class with those properties and then adding that class to those `<img>` tags.